src/client/app/modules/docs/docs.jade

```diff
@@ -1,4 +1,4 @@
 div.row
-  div.doc-list(ui-view="doc-list")
-  div.current-doc(ui-view="container")
+  div.doc-list.sidebar(ui-view="doc-list")
+  div.current-doc.app-content(ui-view="container")
```

Comment:
same here. Also, don't create redundant class titles if possible.. If your `ui-view` has a class `app-content`, your containing jade files should not have `app-content` at the top.